### ISSUE TITLE
RTL Support

### ISSUE THREAD

**Issue by z85510** [Tapadoo/Alerter#107]
How can I make layout to rtl direction ?

**Comment by kpmmmurphy** [comment]
Hi @z85510,

I've pushed a fix for this in version 2.0.4. You can set the content gravity to achieve this.

```java
 Alerter.create(ExampleActivity.this)
                .setTitle("Alert Title")
                .setText("Alert text...")
                .setContentGravity(Gravity.END)
                .show();
```

Do this resolve your issue?

**Comment by z85510** [comment]
Yes, Thank's.
